src/datascience-ui/ipywidgets/kernel/index.ts

```diff
@@ -197,16 +197,19 @@ function initialize() {
 }
 
 function convertVSCodeOutputToExecutResultOrDisplayData(
-    request: NotebookOutputEventParams
+    cellInfo: CellInfo
 ): nbformat.IExecuteResult | nbformat.IDisplayData {
     // New API
     return {
         data: {
-            [request.mime]: request.value
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Might be a better way to handle this versus the any cast, not fully sure. But this works in a pinch. Values coming in are unknowns now with the new API.